doc/development_process/api_lifecycle.rst

```diff
@@ -198,7 +198,11 @@ The following are the requirements for deprecating an existing API:
   where an API is deprecated sooner.
 - What is required when deprecating:
 
-  - Mark as deprecated
+  - Mark as deprecated. This can be done by using the ``__deprecated`` qualifier
+    on a function declaration (``__DEPRECATED_MACRO`` marker in a macro
```

Comment:
This open paren doesn't have a corresponding close. Should it be a comma instead?

Reply:
Making it a comma would suggest there are three ways to mark something deprecated. The intent of my proposal was to clarify that there are two ways (annotation as deprecated or Kconfig-control of behavior), but that the annotation to use is different for functions and macros.  So it needs the close paren after `definition` before the comma.

Though looking at it again it may need more guidance on when to use the Kconfig path, since I don't see how that marks the API deprecated.

Reply:
still, there is a paren too many in this paragraph...

Reply:
Let me see if I can reword that.

Reply:
Reworded